hash/ibm5170_cdrom.xml

```diff
@@ -64,14 +64,65 @@ license:CC0
 		</part>
 	</software>
 
+	<!-- Source: http://redump.org/disc/34886/ -->
+	<!-- <rom name="Blues Brothers, The (Europe).cue" size="94" crc="22379e3c" sha1="aaa704acbe4dcc90e8153f81227f00a83cbdaadd"/> -->
+	<!-- <rom name="Blues Brothers, The (Europe).bin" size="8979936" crc="346e8d8f" sha1="3b775aedf36fcbc33b6a04159d2c4887ec3d611d"/> -->
+	<software name="5plus1_10">
+		<description>5 Plus One: Pack 10 - The Blues Brothers</description>
+		<year>1994</year>
+		<publisher>Prism Leisure Corporation PLC</publisher>
+		<notes><![CDATA[
+CD-ROM includes
+Complete Game: "The Blues Brothers"
+Demo/Shareware: "Bodycount", "Zone 66", "Save Our Pizzas", "Dangerous Dave", "Casino Slots"
+]]></notes>
+		<info name="region" value="Europe" />
+		<sharedfeat name="platform" value="DOS" />
+
+		<part name="cdrom" interface="cdrom">
+			<diskarea name="cdrom">
+				<disk name="Blues Brothers, The (Europe)" sha1="a9a4f1071e7ed928e73535d9bffcd11ee47ac56f" />
+			</diskarea>
+		</part>
+	</software>
+
+	<!-- Source: http://redump.org/disc/35748/ -->
+	<!-- <rom name="TV Sports Football (Europe).cue" size="93" crc="64a640a0" sha1="fbc6fcb695b38c0792312a23ae3c79a4aab6ee7b"/> -->
+	<!-- <rom name="TV Sports Football (Europe).bin" size="6905472" crc="11800913" sha1="6660a474a49f71485cc5ee802dafca90736c53fc"/> -->
+	<software name="5plus1_25">
+		<description>5 Plus One: Pack 25 - TV Sports Football</description>
+		<year>1994</year>
+		<publisher>Prism Leisure Corporation PLC</publisher>
+		<notes><![CDATA[
+CD-ROM includes
+Complete Game: "TV Sports Football"
+Demo/Shareware: "Ranger Fox", "Monster Bash", "Onesimus", "Turoid", "Peristroka"
+]]></notes>
```

Comment:
Just noticed looking at this (after merge), "Peristroka" should be "Perestroika".

I assume it's this game? https://www.mobygames.com/game/toppler